README.md

```diff
@@ -0,0 +1,73 @@
+
+
+# Ansible Network Collection for Common Code (netcommon)
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/builds?project=ansible-collections%2Fansible.netcommon) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/ansible.netcommon)](https://codecov.io/gh/ansible-collections/ansible.netcommon)
+
+The Ansible ``ansible.netcommon`` collection includes common network content to help automate the management of network devices.
```

Comment:
The httpapi connection plugin is used by security and VMware modules as well so many this line can be more generic